backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/tokens/BpkTypography.kt

```diff
@@ -0,0 +1,125 @@
+    hero64: TextStyle = TextStyle(
+      fontWeight = FontWeight.Bold,
+      fontSize = 64.sp, // TODO - replace with token when available
+      letterSpacing = (-0.02).em, // TODO - replace with token when available
+    ),
+    hero48: TextStyle = TextStyle(
+      fontWeight = FontWeight.Bold,
+      fontSize = 48.sp, // TODO - replace with token when available
+      letterSpacing = (-0.02).em, // TODO - replace with token when available
+    ),
+    heading40: TextStyle = TextStyle(
+      fontWeight = FontWeight.Bold,
+      fontSize = BpkFontSize.Xxxl,
+    ),
+    heading32: TextStyle = TextStyle(
+      fontWeight = FontWeight.Bold,
+      fontSize = BpkFontSize.Xxl,
+    ),
+    heading24: TextStyle = TextStyle(
+      fontWeight = FontWeight.Bold,
+      fontSize = BpkFontSize.Xl,
+    ),
+    heading20: TextStyle = TextStyle(
+      fontWeight = FontWeight.Bold,
+      fontSize = BpkFontSize.Lg,
+    ),
+    heading16: TextStyle = TextStyle(
+      fontWeight = FontWeight.Bold,
+      fontSize = BpkFontSize.Base,
+    ),
+    subheading: TextStyle = TextStyle(
+      fontWeight = FontWeight.Normal,
+      fontSize = BpkFontSize.Xl,
+    ),
+    bodyLongform: TextStyle = TextStyle(
+      fontWeight = FontWeight.Normal,
+      fontSize = BpkFontSize.Lg,
+    ),
+    bodyDefault: TextStyle = TextStyle(
+      fontWeight = FontWeight.Normal,
+      fontSize = BpkFontSize.Base,
+    ),
+    footnote: TextStyle = TextStyle(
+      fontWeight = FontWeight.Normal,
+      fontSize = BpkFontSize.Sm,
+    ),
+    caption12: TextStyle = TextStyle(
+      fontWeight = FontWeight.Normal,
+      fontSize = BpkFontSize.Xs,
+    ),
+    label16: TextStyle = TextStyle(
+      fontWeight = FontWeight.Bold,
+      fontSize = BpkFontSize.Base,
+    ),
+    label14: TextStyle = TextStyle(
+      fontWeight = FontWeight.Bold,
+      fontSize = BpkFontSize.Sm,
+    )
+  ) : this(
+    hero64 = hero64.withDefaultFontFamily(defaultFontFamily),
+    hero48 = hero48.withDefaultFontFamily(defaultFontFamily),
+    heading40 = heading40.withDefaultFontFamily(defaultFontFamily),
+    heading32 = heading32.withDefaultFontFamily(defaultFontFamily),
+    heading24 = heading24.withDefaultFontFamily(defaultFontFamily),
+    heading20 = heading20.withDefaultFontFamily(defaultFontFamily),
+    heading16 = heading16.withDefaultFontFamily(defaultFontFamily),
+    subheading = subheading.withDefaultFontFamily(defaultFontFamily),
+    bodyLongform = bodyLongform.withDefaultFontFamily(defaultFontFamily),
+    bodyDefault = bodyDefault.withDefaultFontFamily(defaultFontFamily),
+    footnote = footnote.withDefaultFontFamily(defaultFontFamily),
+    caption = caption12.withDefaultFontFamily(defaultFontFamily),
+    label16 = label16.withDefaultFontFamily(defaultFontFamily),
+    label14 = label14.withDefaultFontFamily(defaultFontFamily),
+  )
+}
+
+private fun TextStyle.withDefaultFontFamily(default: FontFamily): TextStyle {
```

Comment:
It'd be more easier for me if we'd just inlined it to single copy call with default fontFamily.